snake-ladder-ui/src/Components/Board/Board.js

```diff
@@ -1,45 +1,72 @@
-import React, { useRef, useState } from "react";
+import React, { useRef, useState, useEffect } from "react";
 import ReactDice from "react-dice-complete";
-import axios from "axios";
 import "./Board.css";
+import { over } from "stompjs";
+import SockJS from "sockjs-client";
 import baseURL from "../../config";
 
+var stompClient = null;
 const Board = () => {
   //dice
   const diceRef = useRef();
   const [diceValues, setDiceValues] = useState([]);
 
-  const storedData = localStorage.getItem("gameData");
+  const storedData = localStorage.getItem("gameDataNEW");
   const parsedData = JSON.parse(storedData);
-  // console.log(parsedData);
+  const noOfDices = parsedData.board.dice.count;
+  console.log("Parsed Data Log->", parsedData);
   const gameId = parsedData?.id;
   const emailId = parsedData?.emailId;
+  useEffect(() => {
+    myMethod();
+  }, []);
+
+  const myMethod = () => {
+    console.log("Inside My Method");
+    if (!stompClient) {
+      const url =  `${baseURL}/SnakeLadder`;
+      console.log(url);
+      let Sock = new SockJS(url); //server connection
+      stompClient = over(Sock);
+      stompClient.connect({}, onConnected, onError);
+    }
+  };
+  
+  const onConnected = () => {
+    stompClient.subscribe("/movePlayerAll/public", onMovePlayer);
+  };
+
+  const onError = (err) => {
+    console.log(err);
+  };
+ 
 
   const handleRoll = () => {
     diceRef.current.rollAll();
+
+    const movePlayerReq = {
+      gameId: gameId,
+      emailId: emailId,
```

Comment:
This emailId will fetch the mailId of the person who created the game right because it came from getGame api call?
But here we want to send the mail of the person who rolled the dice and not the person who created the game?